content/minfraud/release-notes/2024.mdx

```diff
@@ -7,6 +7,42 @@ title: minFraud Release Notes
 Subscribe to the [minFraud release notes RSS feed](/minfraud/release-notes/rss.xml).
 </Alert>
 
+<ReleaseNote date="2024-04-08" title="Email first seen can be used as a parameter for custom rules">
+
+minFraud Insights and Factors customers can now use the
+`/email/first_seen output` in minFraud custom rules.
```

Comment:
I'd be a nice if this linked to the dev docs: https://dev.maxmind.com/minfraud/api-documentation/responses/#schema--response--email